recipes/xeus-wren/meta.yaml

```
@@ -0,0 +1,58 @@
+{% set name = "xeus-wren" %}
+{% set version = "0.1.2" %}
+{% set sha256 = "bed8835be599b1070eae44abf25f8a68f6fb4a0510e795975a25158928719dba" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/DerThorsten/xeus-wren/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+    - make  # [unix]
+  host:
+    - cppzmq >=4.6.0,<5
+    - zeromq
+    - nlohmann_json >=3.2.0,<3.10
+    - wren >=0.4.0
+    - xeus >=2.0
+    - xtl >=0.7
+
+test:
+  commands:
+    - test -f ${PREFIX}/bin/xwren                                                           # [unix]
+    - test -f ${PREFIX}/share/jupyter/kernels/xwren/kernel.json                             # [unix]
+    - if not exist %LIBRARY_BIN%\\xwren.exe (exit 1)                                        # [win]
+    - if not exist %LIBRARY_PREFIX%\\share\\jupyter\\kernels\\xwren\\kernel.json (exit 1)   # [win]
+    - test -d ${PREFIX}/include/xeus-wren                                                   # [unix]
+    - test -f ${PREFIX}/include/xeus-wren/xinterpreter.hpp                                  # [unix]
```

```suggestion
    - test -f ${PREFIX}/include/xeus-wren/xinterpreter.hpp                                  # [unix]
```

Why do you want to test whether it's a directory?